docs/io.cozy.bills.md

```diff
@@ -16,6 +16,15 @@
 
 ### Optional attributes (but some are important depending the context)
 
+- `recurring`: {String} : is the bill a recurring one ?
+    - `weekly`: the bill is recurring every week in the day of the week corresponding to the `date` attribute.
+    - `monthly`: the bill is recurring every month in the day corresponding to the `date` attribute.
```

Comment:
J'suis pas un grand fan de cette magie. Est-ce qu'on ne devrait pas stocker un objet du genre, histoire d'avoir quelque chose de plus parlant ?  
```
recurring: {
  type: 'monthly',
  day: 24
}

recurring: {
 type: 'yearly',
...
}
```

J'ai par exemple le cas de facture récurrente mensuelle (abo deezer) qui ne sont pas débitées tout le temps le même jour dans le mois, comment va-t-on gérer cela ?

Reply:
I think since the `date` should not be part of the `recurring` object. We can use the `date` as an hint in a case when we are trying to match bills to a recurrence object (separate object in database).

Reply:
Christophe, could you add in your description sample usecases of using recurrence and how this attribute should be used ?

In Banks, we will use `recurrence` inside operations and have a io.cozy.banks.recurrence object linked to an operation that has been found as recurrent. If we find a linked bill for an operation, the `recurrence` object in the `io.cozy.bills` could be used as a hint that the `io.cozy.bank.operations` should be matched to a `io.cozy.banks.recurrence` object with the same frequency.

I am not sure in this usecase that the `date` is important. Since it is already in the bill, I am not sure that it should also be present in the `recurrence` attribute.

Reply:
Sorry, that wasn't clear. When I mention a date attribute, I am referring to the date attribute of the bill itself.

Reply:
But actually, the best reference for the date will be the associated bank operation itself. I wonder if I should even mention how the recurring attribute will be used.

I think I should actually explain what this recurring attribute means. What do you think ?